k8s/prod/common/ccd/data-store-api.yaml

```diff
@@ -28,6 +28,8 @@ spec:
       environment:
         DATA_STORE_DB_HOST: ccd-data-store-api-postgres-db-v11-prod.postgres.database.azure.com
         IDAM_USER_URL: "https://hmcts-access.service.gov.uk"
+        IDAM_API_URL: https://idam-api.platform.hmcts.net
```

Comment:
`IDAM_API_URL` at this moment might cause an issue as Spring might take precedence of this env variable over this https://github.com/hmcts/ccd-data-store-api/blob/master/src/main/resources/application.properties#L64-L66